nms/packages/magmalte/app/views/subscriber/__tests__/SubscriberAddEditTest.js

```diff
@@ -47,6 +47,7 @@ const subscribersMock = {
     name: 'subscriber0',
     active_apns: ['apn_0'],
     id: 'IMSI00000000001002',
+    allowed_nw_types: ['NT_EPC', 'NT_5GC'],
```

Comment:
Replace these with reference to `NETWORK_TYPE` // `NetworkType`

Reply:
`const coreNetworkTypes = Object.keys(CoreNetworkTypes).map(key => CoreNetworkTypes[key]);`
I'm referencing it from this.
`allowed_nw_types: coreNetworkTypes`